selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/intelligent/CommandsEditor.java

```diff
@@ -411,27 +478,52 @@ public void waitTextIntoMacrosContainer(final String expectedText) {
                 webDriver -> getAllVisibleTextFromMacrosContainer().contains(expectedText));
   }
 
+  /**
+   * Gets all visible text from proposal area of the "Command Macros" form.
+   *
+   * @return visible text from proposal area
+   */
   public String getAllVisibleTextFromMacrosContainer() {
```

Comment:
It seems method name could be rephrase to `getVisibleTextFromMacrosContainer` without loss of meaning.